environment-dev.yml

```
@@ -31,5 +31,5 @@ dependencies:
   #
   - pip>=21.1
   - pip:
-      - "-e ."
+      - " .[dev]"
```

```suggestion
      - "-e .[dev]"
```
`-e` is for editable, important for development